runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -5,6 +5,8 @@ authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
 [dependencies]
+wasmer-runtime = { version = "=0.17.1", features = ["default-backend-singlepass"], default-features = false }
```

Comment:
Why transitive dependency via runner doesn't work?

Reply:
I'm not sure, but I got complaints about it when trying to compile.  I'll try to remove it.